content/en/blog/2025/ambient-multicluster/index.md

```diff
@@ -0,0 +1,106 @@
+The outer HBONE connection encrypts cross-cluster traffic, encrypts the destination service FQDN, and allows the source ztunnel and east-west gateway to verify each other's identity.
+The inner HBONE connection encrypts traffic end-to-end, which allows the source ztunnel and destination ztunnel to verify each other's identity.
+Put together, the two HBONE layers stop unauthenticated access, protect against data sniffing, and allow identity verification at every step.
+At the same time, the HBONE layers allow ztunnel to effectively reuse cross-cluster connections, minimizing TLS handshakes.
+
+One drawback is that we encrypt application data twice (once for the outer HBONE and once for the inner HBONE).
+We found this to be an acceptable drawback because it allows us to stick with open standards, and we expect the extra encryption to be negligible compared to the cost of sending data across clusters.
+
+{{< image link="./mc-ambient-traffic-flow.png" caption="Istio ambient multicluster traffic flow" >}}
+
+### Service discovery and scope
+
+Once we have securely connected our clusters, we enable cross-cluster communication for a service by marking it global.
+When a service becomes global, istiod will configure east-west gateways to accept and route traffic destined to the global service.
+Istiod will also read remote apiservers and configure ztunnel with the number of pods for the global service per remote cluster.
+Ztunnel can then proxy traffic to the global service across clusters.
+
+Mesh administrators define the label-based criteria for global services via the `ServiceScope` API,
+and app developers opt into global behavior by labeling their services accordingly.
+The default `ServiceScope` is
+
+{{< text yaml >}}
+=======
+serviceScopeConfigs:
+  - servicesSelector:
+      matchExpressions:
+        - key: istio.io/global
+          operator: In
+          values: ["true"]
+    scope: GLOBAL
+{{< /text >}}
+
+meaning that any service with the `istio.io/global=true` label is global.
+Although the default value is straightforward, the API is flexible and can express complex conditions using a mix of ANDs and ORs.
+
+By default, ztunnel will load balance traffic uniformly across all endpoints --even remote ones--, but this can be configured using the service's `trafficDistribution` field to only cross cluster boundaries when there are no local endpoints.
+Thus, users have control over whether and when traffic crosses cluster boundaries.
+
+## Limitations and Roadmap
+
+Although the current implementation of ambient multicluster has the foundational features for a multicluster implementation,
+there is still a lot of work to be done.
+
+For example, currently, we require that global services, attached waypoints, and `ServiceScope` configuration have uniform configuration across all clusters.
+This greatly simplified our alpha implementation, but we are looking to allow for more configuration skew.
```

Comment:
Remember: we're not giving a report or presenting an academic paper. The readers don't really care about what our experience was building this; they want to know how we're adding value to them.